skbio/stats/composition.py

```diff
@@ -608,10 +611,340 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0-dev")
+def ancom(table, grouping,
+          alpha=0.05,
+          tau=0.02,
+          theta=0.1,
+          multiple_comparisons_correction=None,
+          significance_test=None):
+    r""" Performs a differential abundance test using ANCOM.
+
+    This is done by calculating pairwise log ratios between all features
+    and performing a significance test to determine if there is a significant
+    difference in feature ratios with respect to the variable of interest.
+
+    In an experiment with only two treatments, this test tests the following
+    hypothesis for feature :math:`i`
+
+    .. math::
+
+        H_{0i}: \mathbb{E}[\ln(u_i^{(1)})] = \mathbb{E}[\ln(u_i^{(2)})]
+
+    where :math:`u_i^{(1)}` is the mean abundance for feature :math:`i` in the
+    first group and :math:`u_i^{(2)}` is the mean abundance for feature
+    :math:`i` in the second group.
+
+    Parameters
+    ----------
+    table : pd.DataFrame
+        A 2D matrix of strictly positive values (i.e. counts or proportions)
+        where the rows correspond to samples and the columns correspond to
+        features.
+    grouping : pd.Series
+        Vector indicating the assignment of samples to groups.  For example,
+        these could be strings or integers denoting which group a sample
+        belongs to.  It must be the same length as the samples in `table`.
+        `grouping` must be indexed by the IDs in `table` (i.e., the row labels
```

Comment:
This wording is copied from anosim/permanova and doesn't make sense here because it was written for a `DistanceMatrix` and `grouping`. Can you update to this instead?

```
Vector indicating the assignment of samples to groups.  For example, these could be strings or integers denoting which group a sample belongs to.  It must be the same length as the samples in `table`. The index must be the same on `table` and `grouping` but need not be in the same order.
```